runtime/backtrace_byt.c

```diff
@@ -84,6 +87,8 @@ struct ev_info {
   int ev_endchr;
 };
 
+*/
```

Comment:
I think you should remove the old definition instead of keeping it around as commented dead code.